NitroxClient/MonoBehaviours/Gui/Settings/SettingsOptions.cs

```diff
@@ -0,0 +1,103 @@
+﻿using NitroxClient.Unity.Helper;
+using NitroxModel.Logger;
+using Steamworks;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace NitroxClient.MonoBehaviours.Gui.Settings {
```

Comment:
Please audit your entire PR for formatting (in this case { needs to be on a new line)